source/Symbol/SwiftASTContext.cpp

```diff
@@ -1369,15 +1370,21 @@ static void printASTValidationInfo(
     LLDB_LOG(log, "  -- {0}", ExtraOpt);
 }
 
+void SwiftASTContext::DiagnoseWarnings(Process &process, Module &module) const {
```

Comment:
Process seems unused here?

Reply:
thanks, that's a holdover from an earlier attempt.

Reply:
Sorry.. it's unused in DeserializeCompilerFlags. It's very much in use here :-)